docs/content/standards/deepbook.mdx

```diff
@@ -9,20 +9,15 @@ DeepBook is a next-generation decentralized central limit order book (CLOB) buil
 
 ## Versioning
 
-The latest development iteration of DeepBook is version 3 (DeepBookV3). [DeebBookV3](./deepbookv3.mdx) is currently available in both Devnet and Testnet. [DeepBookV2](./deepbookv2.mdx) is the current Mainnet iteration of DeepBook. 
-
-DeepBookV3 is scheduled to replace DeepBookV2 on Mainnet in the second half of 2024. 
+The latest development iteration of DeepBook is version 3 (DeepBookV3). [DeepBookV3](./deepbookv3.mdx) is currently available across all Sui networks. [DeepBookV2](./deepbookv2.mdx) is currently still available, as well, to provide time for legacy contracts to migrate to DeepBookV3. 
 
 <DocCardList/>
 
 ## Open source
 
 DeepBook is open for community development. You can use the [Sui Improvement Proposals](https://github.com/sui-foundation/sips?ref=blog.sui.io) (SIPs) process to suggest changes to make DeepBook better.
 
-- [DeepBookV2 packages](https://github.com/MystenLabs/sui/tree/main/crates/sui-framework/packages/deepbook) (part of Sui framework)
-- [DeepBookV3 repository on GitHub](https://github.com/MystenLabs/deepbookv3)
-
-
 ## Related links
 
 - <a href="/references/framework/deepbook" data-no-link-check>DeepBook framework docs</a>: Autogenerated framework documentation for DeepBookV2.
+- [DeepBookV3 repository on GitHub](https://github.com/MystenLabs/deepbookv3): DeepBookV3 repository until it's integrated into the Sui framework.  
```

Comment:
Can remove "until it's integrated into the Sui framework." since we don't plan on doing this

Reply:
Also move this reference in front of the DeepBookV2 reference then looks good!